.gitignore

```diff
@@ -0,0 +1,7 @@
+#IntelliJ
```

Comment:
the ignore file is not good, the generated `.iml` files from the IntelliJ are not ignored.. What about if we go with default from the spring initz:

```
.gradle
/build/
!gradle/wrapper/gradle-wrapper.jar

### STS ###
.apt_generated
.classpath
.factorypath
.project
.settings
.springBeans
.sts4-cache

### IntelliJ IDEA ###
.idea
*.iws
*.iml
*.ipr
/out/

### NetBeans ###
/nbproject/private/
/nbbuild/
/dist/
/nbdist/
/.nb-gradle/
```